### DISCUSSION TITLE
[source-mongodb] support for standalone instances

### DISCUSSION THREAD

**Discussion by losblancoo** [airbytehq/airbyte#47285] [Connector Questions]
#### Connector Name
source-mongodb

#### Connector Version
every version > 1.0.0

#### Topic
Every version prior 1.0.0 had support for standalone instances screenshot atached.
<img width="575" alt="Screenshot 2024-10-23 at 11 01 03" src="https://github.com/user-attachments/assets/7b034344-d6d8-450f-9c09-ec0fb8ee8d8a">

Every version > 1.0.0 has support only for replica sets and mongo atlas.
<img width="575" alt="Screenshot 2024-10-23 at 11 02 42" src="https://github.com/user-attachments/assets/7f8201d5-6324-4aaa-8992-ce1fd61add96">

Are there plans for reintroducing support for standalone instance types? Also what was the reason for removing this feature?

**Comment by briankariuki** [comment]
I believe you can still use a single instance replica set. You have to enable the replication feature in your mongod.conf

**Comment by losblancoo** [comment]
And what if we use https://www.mongodb.com/docs/manual/core/sharded-cluster-components/ sharded mongo cluster? Cause we cant connect to it using replica set option. In our sharded cluster we have > 10 replica sets. And we used to connect to it **standalone mongodb instance** option.

[a comment by an automated account is not shown]
